app/Actions/SportAction.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Actions;
+
+use App\Http\Requests\SportRequest;
+use App\Models\Country;
+use App\Models\Sport;
+
+class SportAction
+{
+    public function store(SportRequest $request)
+    {
+        foreach ($request->sport as $sport_id => $places) {
+
+            $sport = Sport::findOrFail($sport_id);
```

Comment:
@sergeev-k this will fire a database query each time. What if there are 20 sports in the form? It would be better to get all sports into collection upfront, and then find data from there